src/test/java/com/google/api/codegen/testdata/python_README_library.baseline

```diff
@@ -2,7 +2,9 @@
 Python Client for Google Example Library API (`Alpha`_)
 ==================================================================================================
 
-Idiomatic Python client for `Google Example Library API`_
+Idiomatic Python client for `Google Example Library API`_:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I was thinking it would look better to go from `h1` to `h3` on a purely subjective stylistic level. If you think it should be dashes that's fine!